src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/managedexports.cpp

```diff
@@ -216,6 +217,7 @@ http_get_application_properties(
 
     auto const serverAddresses = BindingInformation::Format(pConfiguration.QueryBindings(), pInProcessApplication->QueryApplicationVirtualPath());
     pIISCofigurationData->pwzBindings = SysAllocString(serverAddresses.c_str());
+    pIISCofigurationData->maxRequestBodySize = pInProcessApplication->QueryConfig().QueryMaxRequestBodySizeLimit();
```

Comment:
`pIISCofigurationData`

![A man pauses for a second and then his eye twitches](https://media.giphy.com/media/s5mOs2Va2FE6k/giphy.gif)

Reply:
I won't make you change it and cycle another build if people approve this as-is, but if you go back in to this code, a rename to fix the typo would be nice ;P

Reply:
`if(`

Reply:
I have a problem. But I also have a [solution?](https://github.com/aspnet/Extensions/pull/1654)